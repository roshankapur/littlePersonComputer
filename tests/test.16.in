2018
3117
4211
2018
3016
2118
1019
2020
3019
2120
4000
2020
3218
2121
1121
4300
1
5
0
-99999
30
10
20
40
50