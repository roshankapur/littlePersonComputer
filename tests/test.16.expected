***Start LPC Execution***
Contents of 21: 30
***End LPC Execution***
